client/src/components/Reservation/Reservation.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import $ from 'jquery';
+
+import Calendar from '../Reservation/Calendar.jsx';
+import People from '../Reservation/People.jsx';
+import Time from '../Reservation/Time.jsx';
+
+import Modal from '../Modal/Modal.jsx';
+
+class Reservation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: new Date(),
+    };
+    this.popupModal = this.popupModal.bind(this);
+  }
+
+  componentDidMount() {
+    const modal = $('.modal');
+    $('.show-modal').click(() => {
+      modal.fadeIn();
+    });
+
+    $('.close-modal').click(() => {
+      modal.fadeOut();
+    });
+  }
+
```

Comment:
Does your modal render by fading in and then immediately out when the component mounts?
It seems unnecessary to invoke the fadeIn and fadeOut function on componentDidMount. Could this functionality be moved?